pkg/storage/storage.go

```diff
@@ -77,9 +77,10 @@ type coreStorage struct {
 	Storage
 	regionStorage endpoint.RegionStorage
 
-	useRegionStorage int32
-	regionLoaded     bool
-	mu               syncutil.Mutex
+	useRegionStorage        int32
+	regionLoadedFromDefault bool
+	regionLoadedFromStorage bool
```

Comment:
Why do we need to distinguish these two kinds?

Reply:
I want to avoid problems after ps.useRegionStorage switch

Reply:
How about using string? 

`regionLoadedFrom`: "", "etcd", "leveldb"

Reply:
I don't think so, we just have these two ways now